internal/core/repository/src/main/kotlin/com/recco/internal/core/repository/RecommendationRepository.kt

```diff
@@ -23,7 +25,12 @@ import javax.inject.Inject
 class RecommendationRepository @Inject constructor(
     private val api: RecommendationApi
 ) {
-    private val supportedContentTypes = listOf(ContentTypeDTO.ARTICLES, ContentTypeDTO.QUESTIONNAIRES)
+    private val supportedContentTypes = listOf(
```

Comment:
`ContentTypeDTO.QUESTIONNAIRES` only has to be sent for the tailored sections, we don't want to show questionnaires in any other section. Is that the case?

Reply:
Correct